emailpassword/docs/common-customizations/sessions/user-information-front-end.md

```diff
@@ -49,6 +49,7 @@ render() {
 }
 ```
 
+You can read more about `EmailPasswordAuth` in [its API guide](/docs/auth-react/docs/emailpassword/email-password-auth)
```

Comment:
I suggest that you understand what the current recipe docs say about these auth wrappers, and where all it's mentioned.

Then you should think about how best to explain everything about these auth wrappers. You want to talk about:
- Protecting a website route
- Checking if a user is logged in
- Fetching session info on the frontend (this page)
- Handling session expiry
- Anything else?

Once you have a clear picture of what needs to be communicated, then you can see the current pages that exist. If you can add the content to the current pages itself, then cool, else add new pages.

If you'd like to remove current pages and rewrite them entirely, that's an option too.

Reply:
Exactly those usecases are covered in `SessionAuth` docs. I think the usecases you mentioned should be described in `auth-react` docs, since like right now we added a feature, which added one usecase (session expiry). 

I can revisit this page to contain the usecases, but it's going to differ between SDK versions.

One solution I can think about is just mention few of them (protecting website route/component, providing session data to child components) and link to the API.

Reply:
> One solution I can think about is just mention few of them (protecting website route/component, providing session data to child components) and link to the API.

This works. The way we have structured our docs right now is that all "guides" / "how tos" are in the recipe docs, and the SDK docs are only to see what the available functions are.